README.md

```diff
@@ -111,50 +116,18 @@ _Happy Coding!_
+We'd love for you to contribute to our source code and to make **Webex Components** even better than it is today!
+Here are some [guidelines](https://github.com/webex/components/blob/master/CONTRIBUTING.md) that we'd like you to follow.
 
 ### Issues
 
 Please open an [issue](https://github.com/webex/components/issues) and we will get to it in an orderly manner.
 Please leave as much as information as possible for a better understanding.
 
-### Contributing Code
-
-We are using [Webex Style Guide eslint rule](https://github.com/webex/web-styleguide/tree/master/packages/node_modules/%40webex/eslint-config-react) and [prettier](https://github.com/prettier/prettier) to lint the code style.
-You can "prettify" your code before committing via:
-
-```bash
-npm run prettier:write
-```
-
-### Release Process
-
-There is a list of commit types provided [here](https://github.com/webex/components/blob/master/CONTRIBUTING.md#type). However, not all commits trigger our release process.
-We are using [semantic-release](https://github.com/semantic-release/semantic-release) to fully automate the version management and package publishing.
-By default `semantic-release` uses the [Angular Commit Message Conventions](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines) and triggers release and publishing based on the following rules:
-
-| Commit                             | Release type  |
-| ---------------------------------- | ------------- |
-| Commit with type `BREAKING CHANGE` | Major release |
-| Commit with type `feat`            | Minor release |
-| Commit with type `fix`             | Patch release |
-| Commit with type `perf`            | Patch release |
-
-#### Commit linter
-
-We are using [commitlint](https://github.com/conventional-changelog/commitlint) to lintify the commit messages.
-Please make sure to choose the appropriate commit [type](https://github.com/webex/components/blob/master/CONTRIBUTING.md#type), [scope](https://github.com/webex/components/blob/master/CONTRIBUTING.md#scope) and [subject](https://github.com/webex/components/blob/master/CONTRIBUTING.md#scope).
-
 ## License
 
 [MIT License](https://opensource.org/licenses/MIT)
 
 ## Support
 
 For more developer resources, tutorials and support, visit the Webex developer portal, https://developer.webex.com.
-
-## Team
-
-| [![Adam Weeks](https://github.com/adamweeks.png?size=100)](https://github.com/adamweeks) | [![Arash Koushkebaghi](https://github.com/akoushke.png?size=100)](https://github.com/akoushke) | [![Lalli Flores](https://github.com/lalli-flores.png?size=100)](https://github.com/lalli-flores) | [![Timothy Scheuering](https://github.com/InteractiveTimmy.png?size=100)](https://github.com/InteractiveTimmy) | [![David Hoff](https://github.com/harborhoffer.png?size=100)](https://github.com/harborhoffer) |
```

Comment:
![img](https://media0.giphy.com/media/Ru9sjtZ09XOEg/giphy.gif)

Reply:
When over half the team dips 😭